index.js

```diff
@@ -4,7 +4,29 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    console.log(process.argv)
+    if (process.argv[2] === undefined)
+    return `No operation provided...`
+    if (!(process.argv.includes("plus") || process.argv.includes("minus"))) {
+     return `Invalid operation: ${process.argv[2]}`
+    }
+    if (process.argv.length >= 4) {
+    let sum = Number(process.argv[3]) // equal first number
```

Comment:
Since you can subtract and add a more accurate variable name would be something like total or result